### ISSUE TITLE
How do I use this without Java?

### ISSUE THREAD

**Issue by scrptktty** [cezarykluczynski/stapi#5]
I would love to use your API for my first ever programming project! 

I could not find a resource with the end point url or REST url. All of the tools listed are Java oriented. I am currently learning JavaScript. 

For example, I found this [link](url) in the code, but that URL just the main page. 

The path I am looking for is for the characters.

I am betting it will be something like `characters/entity/Janeway.yaml` but I cannot figure out the domain. What server should I make the request to?

tl:dr can you show me a curl command to get Janeway's info like gender or DOB. 

Looking forward to you help! Thank you.

**Comment by scrptktty** [comment]
If you are curious, this is what I am working on atm: https://runkit.com/scrptktty/star-trek-gender.

**Comment by cezarykluczynski** [comment]
Hello.

STAPI cannot be currently queried using JavaScript in browser, because of unconfigured CORS policy. This can be bypassed using plugin like [this one](https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi?utm_source=chrome-app-launcher-info-dialog) or anything that sets fake CORS headers. I'll try to fix that in the near future, so plugin won't be needed. If you are using node.js, CORS limitation does not apply.

All available services can be found [here](http://stapi.co/api/). REST services are on the bottom. Every service has the same layout: /character?uid=UIDHERE for querying for a single entity, and /character/search for searching.

To access characters, go to url like this one: http://stapi.co/api/v1/rest/character/search?pageNumber=0&pageSize=100

Then, just increment pageNumber until you reach a result that has page.lastPage property set to true, or a page that is empty. [This](http://stapi.co/api/v1/rest/character/search?pageNumber=59&pageSize=100) is the last page now, and the [next page](http://stapi.co/api/v1/rest/character/search?pageNumber=0&pageSize=100) is empty. The number of result can change in the future, so do not assume there will always be 59 pages of characters.

For the performers, do the same, but change character to performer in the URL, like this: http://stapi.co/api/v1/rest/performer/search?pageNumber=0&pageSize=100

To get Janeway specifically, you need to look her up using POST request. Just post to http://stapi.co/api/v1/rest/character/search?pageNumber=0&pageSize=100 with name set to Kathryn Janeway. [Postman](https://chrome.google.com/webstore/detail/postman/fhbjgbiflinjbdggehcddcbncdddomop) is a nice Chrome plugin to play around with REST APIs.
